app/sidekiq/lighthouse/submit_benefits_intake_claim.rb

```diff
@@ -149,5 +150,13 @@ def cleanup_file_paths
       Common::FileHelpers.delete_file_if_exists(@pdf_path) if @pdf_path
       @attachment_paths&.each { |p| Common::FileHelpers.delete_file_if_exists(p) }
     end
+
+    def check_zipcode(address)
+      if address['country'].upcase.in?(%w[USA US])
+        [address['postalCode'], true]
+      else
+        [FOREIGN_POSTALCODE, false]
```

Comment:
the metadata validator handles the FOREIGN_POSTALCODE for you, this is duplicate logic.

Reply:
Gotcha